protos/flyteidl/event/event.proto

```diff
@@ -75,6 +87,11 @@ message ParentTaskExecutionMetadata {
     core.TaskExecutionIdentifier id = 1;
 }
 
+message ParentNodeExecutionMetadata {
+    // Unique identifier of the parent node id within the execution
+    string id = 1;
```

Comment:
this references the `node_id`?

do we anticipate this changing over time ever? would it make sense to make it a proper proto message with just one field for now?

Reply:
This is the unique_node_id (we are moving from node_id to unique node id). I am open to suggestions. Merely following the `ParentTaskExecutionMetadata` model. But fine with anything.

Reply:
what is the `unique_node_id` -> Is it not the nodeexecutionIdentifier?

Reply:
@kumare3 It is the id in `nodeexecutionIdentifier `.

nodeexecutionIdentifier has node_id. nodeexecutionIdentifier also has WorkflowExecutionIdentifier- which project,domain, name

Reply:
then should this not be an object / structure?

Reply:
That's what I mentioned to @katrogan. I do not want that. It will only cause unwanted project, domain, name checks. 

You can't have a parent node from a different execution.